### PR TITLE
[Roles api] Roles api v2 documentation

### PR DESCRIPTION
**Pull request by ssc3**
### What does this PR do?

This modifies roles api documentation to display roles v2 api

### Preview link

https://docs-staging.datadoghq.com/shubho/roles-v2/account_management/rbac/role_api/

### CODE REVIEW

File: content/en/account_management/rbac/role_api.md [hunk 1]
```diff
@@ -17,162 +18,375 @@ The Roles API can be used to create and manage Datadog roles, what global permis
...
+##### Example:
 
-Example:
 
 ```sh
-curl -X GET "https://app.datadoghq.com/api/v1/roles?api_key=${API_KEY}&application_key=${APP_KEY}"
+curl -X GET "https://app.datadoghq.com/api/v2/roles" -H "DD-API-KEY: ${API_KEY}" -H "DD-APPLICATION-KEY: ${APP_KEY}"
+
 
 # Response:
```
Comment (pietdaniel) [inline content/en/account_management/rbac/role_api.md]: can we move this to its own formatted section with 
```
```json
```
as its header

File: content/en/account_management/rbac/role_api.md [hunk 2]
```diff
@@ -17,162 +18,375 @@ The Roles API can be used to create and manage Datadog roles, what global permis
...
+#                             "id": $PERMISSION_UUID
+#                         }
+#                     ]
+#                 }
+#             }
+#         }
 # }
 ```
 
 ### Create Role
```
Comment (pietdaniel) [inline content/en/account_management/rbac/role_api.md]: does this return the newly created role as the response? if so we should include this

Comment (ssc3) [reply content/en/account_management/rbac/role_api.md]: Done

File: content/en/account_management/rbac/role_api.md [hunk 3]
```diff
@@ -17,162 +18,548 @@ The Roles API can be used to create and manage Datadog roles, what global permis
...
 Description: Returns all roles, including their names and uuids.  
 Method: GET  
-Endpoint: `api/v1/roles`  
+Endpoint: `api/v2/roles`  
 Required Payload: No Payload
 
 ##### ARGUMENTS
 
+* **`page[size]`** [*optional*, *default*=**0**]:
+Page Number of roles to return for a given page
```
Comment (l0k0ms) [inline content/en/account_management/rbac/role_api.md]:
```suggestion
Page number of roles to return for a given page.
```

File: content/en/account_management/rbac/role_api.md [hunk 4]
```diff
@@ -17,162 +18,548 @@ The Roles API can be used to create and manage Datadog roles, what global permis
...
-# Response:
-# {
-#  "id": <number>,
-#  "name": <string>,
-#  "uuid": <string>
-# }
+##### Example:
+
+```sh
+curl -X GET "https://app.datadoghq.com/api/v2/roles/$ROLE_UUID" -H "DD-API-KEY: ${API_KEY}" -H "DD-APPLICATION-KEY: ${APP_KEY}"
```
Comment (l0k0ms) [inline content/en/account_management/rbac/role_api.md]:
```suggestion
curl -X GET "https://app.datadoghq.com/api/v2/roles/$ROLE_UUID" -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>"
```

**Pull request merged**